src/hotspot/share/gc/parallel/psPromotionManager.inline.hpp

```diff
@@ -258,8 +254,7 @@ inline oop PSPromotionManager::copy_unmarked_to_survivor_space(oop o,
   ContinuationGCSupport::transform_stack_chunk(new_obj);
 
   // Now we have to CAS in the header.
-  // Make copy visible to threads reading the forwardee.
-  oop forwardee = o->forward_to_atomic(new_obj, test_mark, memory_order_release);
+  oop forwardee = o->forward_to_atomic(new_obj, test_mark, memory_order_relaxed);
```

Comment:
I think this needs a comment similar to what's in G1ParScanThreadState::do_copy_to_suvivor_space, e.g.
https://github.com/openjdk/jdk/blob/f101e153cee68750fcf1f12da10e29806875b522/src/hotspot/share/gc/g1/g1ParScanThreadState.cpp#L504-L507